controllers/duration.js

```diff
@@ -0,0 +1,9 @@
+const log = require('../models/duration');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const log = require("../models/duration");
```